test/server_test.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
         self.unix_socket = os.path.join(self.temp_dir, 'luigid.sock')
 
     def run_server(self):
-        luigi.server.run(unix_socket=unix_socket)
+        luigi.server.run(unix_socket=self.unix_socket)
```

Comment:
Argh https://github.com/spotify/luigi/pull/1174/files#diff-9b595b54a300214f2172cde2e9c1eec4R179 is meant to inherit ServerTestRun!